nethserver-dc.spec

```diff
@@ -6,7 +16,7 @@ Summary:        NethServer Domain Controller configuration
 License:        GPLv3+
 URL: %{url_prefix}/%{name}
 Source0:        %{name}-%{version}.tar.gz
-Source1:        https://github.com/NethServer/ns-samba/releases/download/4.9.13/ns-samba-4.9.13-1.ns7.x86_64.rpm
+Source1:        https://github.com/NethServer/ns-samba/releases/download/4.9.13/ns-samba-4.9.13-1.ns7.%{source1_arch}.rpm
 
```

Comment:
Nothing really somthing wrong but it we have to keep in mind the source-package will differ for different arch's.
IOW you can't rebuild a source package produced on x86_64 on arm. 

IMHO it is good enough for us, for reference the solution how it "should be" :

Download all ns-samba arch sources so they are included in the SPRM.
```
Source1:        https://github.com/NethServer/ns-samba/releases/download/4.8.12/ns-samba-4.8.12-1.ns7.x86_64.rpm
Source2:        https://github.com/NethServer/ns-samba/releases/download/4.8.12/ns-samba-4.8.12-1.ns7.aarch64.rpm
Source3:        https://github.com/NethServer/ns-samba/releases/download/4.8.12/ns-samba-4.8.12-1.ns7.arm7hl.rpm
```


then @ `%install ` do something like this:

```
%ifarch %{arm}
mv %{SOURCE3} root/usr/lib/nethserver-dc
%elif
%ifarch %{aarch64}
mv %{SOURCE2} root/usr/lib/nethserver-dc
%else
mv %{SOURCE1} root/usr/lib/nethserver-dc
%endif
```

But this means arm variants od ns-samba **must**  be available during x86_64 builds.
IMO we cannot take that risk..

Reply:
The RPM within another RPM **is the real issue**. The correct approach IMO is adding another YUM repository for the `ns-samba` package, enabled only during `nsdc` install/upgrades.

Reply:
yes , but we want to be sure it is never installed outside the container. 
that is the reason it is in there isn't it?

Reply:
yes, because it conflicts with upstream `samba` packages. They can't be in the same repo, but I hope nsdc installation can work with an extra repository.